cfgov/data_research/tests/management/commands/test_conference_export.py

```diff
@@ -1,8 +1,13 @@
+from __future__ import unicode_literals
```

Comment:
Also happy to leave conference code untouched by this PR if that would keep concerns separated.